hcloud/exp/kit/actions/actions.go

```diff
@@ -0,0 +1,11 @@
+package actions
```

Comment:
Maybe using actions as package name will conflict with a lot of variables. Maybe actionutils is better?